scripts/airgapped/save-images-to-cache.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+import logging
+
+import docker
+
+from utils import get_images_list_from_file, get_or_pull_image
+
+cli = docker.client.from_env()
+
+log = logging.getLogger(__name__)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Pull locally list of images")
+    parser.add_argument("images")
+    args = parser.parse_args()
+
+    images_ls = get_images_list_from_file(args.images)
+    images_len = len(images_ls)
+    for idx, image in enumerate(images_ls):
+        log.info("%s/%s", idx + 1, images_len)
+        get_or_pull_image(image)
```

Comment:
You are right, that pulling would either way would use the local cache.

I had broken down that function as get and then pull because I wanted to have slightly clear logs that the script used a cached image, because it wasn't that clear previously.

I'd personally prefer the current function, as it will also completely skip going to the internet (as pulling would check the layers iirc). But if you feel strongly about this one I can update it.